### DISCUSSION TITLE
直播画面卡顿

### DISCUSSION THREAD

**Discussion by Genicur** [ywmoyue/biliuwp-lite#868] [bug]
### Bug 描述

观看直播时，会一直闪烁缓冲中，网速正常，在浏览器看直播无问题。

### 操作步骤

```plain text
打开应用，进入直播间，画面闪烁缓冲中，切换线路和画质都无法解决。
```

### 期望结果

正常播放

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

24H2 LTSC

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by Genicur** [comment]
是低延迟模式的问题（

**Comment by ywmoyue** [comment]
不开低延迟模式即可，该issue先转discussion
